### PR TITLE
Fix loading on undefined console error

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

File: src/plugins/addLoadingFlag.js [hunk 1]
```diff
@@ -2,12 +2,13 @@ import { wrap, Chart } from 'highcharts/highstock';
 
 export default () => {
     wrap(Chart.prototype, 'showLoading', function showLoading(proceed, ...args) {
-        this.isLoading = true;
+        if (Object.keys(this).length === 0) return;
```
Comment (borisyankov) [inline src/plugins/addLoadingFlag.js]: Is that the best way to check?
Is this what you meant typeof this === 'undefined'
If so, why does it happen?

Comment (qingweibinary) [inline src/plugins/addLoadingFlag.js]: Not sure, thus I did not update package.json

At some point, `this` is a `{}` when `showLoading` is called
And I do not know why, any clue from your side?

Comment (borisyankov) [inline src/plugins/addLoadingFlag.js]: Nope, but I prefer we know what is going on, than just adding 'if's :)

Comment (borisyankov) [inline src/plugins/addLoadingFlag.js]: I'll merge and debug myself.

Comment (qingweibinary) [inline src/plugins/addLoadingFlag.js]: Additional information

when `showLoading` is called, this plugin code will be trigger twice, the first time is correct, the second is somewhat wrong as `this` became empty object, which throw inside highchart code

**Pull request merged**